components/shared_code/src/shared_data_model/sources/sonarqube.py

```diff
@@ -343,6 +392,7 @@ def violation_entity_attributes(
             name="violation",
             attributes=violation_entity_attributes(include_resolution=True, include_rationale=True),
         ),
+        "todos": VIOLATION_ENTITY,
```

Comment:
But this is `name="violations"`? Weird btw that the one on the next line isn't even using `VIOLATION_ENTITY`

Reply:
SonarQube is reporting rule violations, hence the measurement entities are called violations. Fixed the "next line".